WebApplication/package.json

```
@@ -40,5 +40,17 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@iconify/icons-bx": "^1.1.0",
+    "@iconify/icons-cil": "^1.1.0",
+    "@iconify/icons-emojione": "^1.1.0",
+    "@iconify/icons-emojione-v1": "^1.1.0",
+    "@iconify/icons-jam": "^1.1.0",
+    "@iconify/icons-mdi": "^1.1.0",
+    "@iconify/icons-ri": "^1.1.0",
+    "@iconify/icons-subway": "^1.1.0",
+    "@iconify/react": "^1.1.3",
```

```suggestion
    "@iconify/icons-bx": "^1.1.0",
    "@iconify/icons-cil": "^1.1.0",
    "@iconify/icons-emojione": "^1.1.0",
    "@iconify/icons-emojione-v1": "^1.1.0",
    "@iconify/icons-jam": "^1.1.0",
    "@iconify/icons-mdi": "^1.1.0",
    "@iconify/icons-ri": "^1.1.0",
    "@iconify/icons-subway": "^1.1.0",
    "@iconify/react": "^1.1.3",
```
Remove unused dependencies